maintainers/maintainer-list.nix

```diff
@@ -2700,6 +2700,11 @@
     email = "info+nix@chmist.com";
     name = "karolchmist";
   };
+  kayhide = {
```

Comment:
Please submit this as a separate commit.

Reply:
@mmahut Okay, I have done it! Please take a look.